File: Codechef/Codechef_SEP_CookOff_2020/codechef.01.py
try:
    test = int(input())
    for _ in range(test):
        n, k, l = [int(x) for x in input().split()]
        if k * l < n:
            print("-1")
            continue
        if n > 1 and k == 1:
            print("-1")
            continue
        else:
            count = 0
            i = 1
            while i <= k + 1:
                print(i, end = " ")
                count += 1
                i += 1
                if i == k + 1:
                    i = 1
                if count == n:
                    print("")
                    break
except:
    pass

File: Codechef/Codechef_AUG_LunchTime_2020/codechef.01.cpp
#include<bits/stdc++.h>
using namespace std;

int main()
{
    int test;
    cin >> test;
    while(test--)
    {
        int temp, n;
        cin >> n;

        vector<int> arr;
        
        for(int i = 0; i < n; i++)
        {
            cin >> temp;
            arr.push_back(temp);
        }

        vector<int> freq;
        
        for(int i = 1; i <= 10; i++)
        {
            temp = count(arr.begin(), arr.end(), i);
            freq.push_back(temp);
        }

        int maxElement = *std::max_element(freq.begin(), freq.end());

        vector<int> res;

        for(int i = 1; i <= maxElement; i++)
        {
            temp = count(freq.begin(), freq.end(), i);
            res.push_back(temp);
        }

        int maxElementIndex = max_element(res.begin(),res.end()) - res.begin();

        cout << maxElementIndex + 1 << endl;
    }
return 0;
}